README.md

```diff
@@ -1 +1,12 @@
-# hw01
\ No newline at end of file
+# Hello World: Homework 1  
+> _*Either you run the day or the day runs you.*_
+> - Jim Rohn
```

Comment:
Great quote! (Again, I'd throw in a blank line between this and the _following_ header as well as the preceding, just to get in the practice of doing so.)